packages/core/src/routes/connector/authorization-uri.ts

```diff
@@ -60,13 +62,15 @@ export default function connectorAuthorizationUriRoutes<T extends ManagementApiR
           headers: { userAgent },
         },
         /**
-         * Same as above, passing `notImplemented` only works for connectors not relying on session storage.
-         * E.g. Google and GitHub
+         * Return session storage to api client
          */
-        notImplemented
+        async (storage: ConnectorSession) => {
+          // eslint-disable-next-line @silverhand/fp/no-mutation
+          session = { ...session, ...storage };
+        }
       );
 
-      ctx.body = { redirectTo };
+      ctx.body = { redirectTo, session };
```

Comment:
It seems the client will have to deal with the session object now, which is something we tried to avoid in our initial design, for security reasons.